### DISCUSSION TITLE
How do I go back to previous page?

### DISCUSSION THREAD

**Discussion by ohpaul** [molefrog/wouter#463] [Q&A] [answered]
Hi everyone,

I've been searching for the answer for a while but I couldn't find it anywhere on the docs or the web :(
I was wondering how I could go to the previous page from the current page.

For instance, in `'react-router-dom'`, 

`let history = useHistory();`
`history.goBack()`
is used to go back to the previous page.

If anyone could help me with this, that would be great!

**Comment by molefrog** [comment, marked as answer]
The easiest way (if you are using a `useBrowserLocation` hook which is the default), is to use the native History API: `history.back()` (`history` is a global object).
